demisto_sdk/tests/integration_tests/logger_integration_test.py

```diff
@@ -1,163 +0,0 @@
-import logging
-import shutil
-from pathlib import Path
```

Comment:
why deleted?

Reply:
Changed it to be a unit test, I'll explain offline.

Reply:
changed to unit tests